### DISCUSSION TITLE
Is it possible to set colors manually in the CoveragePlot?

### DISCUSSION THREAD

**Discussion by ericli0419** [stuart-lab/signac#503] [Q&A] [answered]
Hello!

I am wondering how I can set cell type colors manually in the CoveragePlot.

Thanks!

**Comment by timoast** [comment, marked as answer]
Please see https://github.com/timoast/signac/issues/63

**Reply by ericli0419** [reply]
Thank you!

`p <- CoveragePlot(object, region)
p + scale_fill_manual(values = color24)` worked perfectly.
